proto/storage/compliance_operator_v2.proto

```diff
@@ -195,3 +195,26 @@ message ComplianceOperatorProfileClusterEdge {
     string        profile_uid = 3[(gogoproto.moretags) = 'search:"Compliance Profile UID,hidden"'];
     string        cluster_id  = 4[(gogoproto.moretags) = 'search:"Cluster ID,hidden" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
 }
+
+// ComplianceOperatorScanSettingBindingV2 associates a scan setting with one or more compliance profiles
+// Next Tag: 9
+message ComplianceOperatorScanSettingBindingV2 {
+    string                                  id                  = 1[(gogoproto.moretags) = 'sql:"pk,id"'];
+    string                                  name                = 2[(gogoproto.moretags) = 'sql:"index=category:unique"'];
```

Comment:
We can't have a unique constraint on name as multiple clusters will have scan setting bindings of the same name.  But we probably want that field to be searchable though.